include/data/data.h

```diff
@@ -42,7 +42,7 @@
 #define L1_GLOBAL_DATA __attribute__((section(".data_l1")))
 #define L1_DATA L1_GLOBAL_DATA
 
-#ifdef USE_CLUSTER
```

Comment:
USE_CLUSTER should still be used.
The goal of RT_LOCAL_DATA is to map the data of the application either to cluster or to fc depending on where the test is run. If we use ARCHI_HAS_CLUSTER, data will always be put on cluster side.

Reply:
Got it. Just restored that #if statement.